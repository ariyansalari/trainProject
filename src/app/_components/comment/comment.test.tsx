import { render } from "@testing-library/react";
import { Comment } from ".";

describe("comment component", () => {
  test("chages of comment", () => {
    const { container } = render(
      <Comment
        id={0}
        date={""}
        userId={undefined}
        fullName={""}
        commentText={""}
        score={null}
        isResponse={false}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
