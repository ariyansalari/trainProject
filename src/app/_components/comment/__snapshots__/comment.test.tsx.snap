// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comment component chages of comment 1`] = `
<div>
  <div
    class="comment comment-start"
  >
    <div
      class="comment-image"
    >
      <div
        class="avatar 40 avatar-neutral"
        style="width: 40px; height: 40px;"
      >
        <svg
          fill="none"
          height="20"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
          viewBox="0 0 24 24"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M 13.6 7.7 C 14.5 8.5 14.5 10 13.6 10.8 C 12.7 11.7 11.3 11.7 10.4 10.8 C 9.5 10 9.5 8.5 10.4 7.7 C 11.3 6.8 12.7 6.8 13.6 7.7"
          />
          <path
            d="M 16.3 16.5 C 16.1 15.2 14.9 14.1 13.5 14.1 H 10.5 C 9.1 14.1 7.9 15.2 7.7 16.5"
          />
          <path
            clip-rule="evenodd"
            d="M 17 21 H 7 C 4.8 21 3 19.2 3 17 V 7 C 3 4.8 4.8 3 7 3 H 17 C 19.2 3 21 4.8 21 7 V 17 C 21 19.2 19.2 21 17 21 Z"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class="comment-header"
    >
      <time
        class="text-xs opacity-50 mx-2"
      />
    </div>
    <div
      class="comment-bubble "
    />
  </div>
</div>
`;
